buildtest/buildsystem/checks.py

```diff
@@ -316,6 +316,69 @@ def assert_ge_check(builder):
     return all(assert_check)
 
 
+def assert_le_check(builder):
+    """Perform check on assert less than and equal when ``assert_le`` is specified in buildspec. The return is a boolean value that determines if the check has passed.
+    One can specify multiple assert checks to check each metric with its reference value. When multiple items are specified, the operation is a logical AND and all checks
+    must be ``True``.
+
```

Comment:
can you please add the `Args` for `builder` so it shows up in docstring

Reply:
I already added on both assert_ge_check and assert_le_check